source/pidomus.cc

```diff
@@ -531,15 +533,16 @@ void piDoMUS<dim, spacedim, LAC>::refine_mesh ()
   if (adaptive_refinement)
     {
       Vector<float> estimated_error_per_cell (triangulation->n_active_cells());
-      KellyErrorEstimator<dim>::estimate (*dof_handler,
-                                          QGauss < dim - 1 > (fe->degree + 1),
-                                          typename FunctionMap<dim>::type(),
-                                          distributed_solution,
-                                          estimated_error_per_cell,
-                                          ComponentMask(),
-                                          0,
-                                          0,
-                                          triangulation->locally_owned_subdomain());
+      KellyErrorEstimator<dim,spacedim>::estimate (interface.get_mapping(),
+                                                   *dof_handler,
+                                                   QGauss <dim-1> (fe->degree + 1),
+                                                   typename FunctionMap<spacedim>::type(),
+                                                   distributed_solution,
+                                                   estimated_error_per_cell,
+                                                   ComponentMask(),
```

Comment:
Can you open an issue for this component mask? The user should be able to select a component mask for the refinement...